shamir_mnemonic/cli.py

```diff
@@ -99,7 +99,7 @@ def create(scheme, groups, threshold, exponent, master_secret, passphrase, stren
                 "master_secret", f"Secret bytes must be hex encoded"
             ) from e
     else:
-        secret_bytes = os.urandom(strength // 8)
+        secret_bytes = secrets.token_hex(strength // 8)
```

Comment:
secret_bytes should be `token_bytes`

Reply:
Do you mean that I should rename the existing variable?

Reply:
no, you should change the call. secret_bytes are supposed to be bytes, not hex string

Reply:
No, you should be calling `secrets.token_bytes()`.

Reply:
Sorry for my confusion. Done now.